src/wallet/test/coinselector_tests.cpp

```diff
@@ -377,7 +381,8 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
         expected_result.Clear();
         add_coin(10 * CENT, 2, expected_result);
         CCoinControl coin_control;
-        const auto result11 = SelectCoins(*wallet, available_coins, /*pre_set_inputs=*/{}, 10 * CENT, coin_control, coin_selection_params_bnb);
+        CAmount fee_to_deduce = available_coins.coins[OutputType::BECH32].begin()->GetFee();
+        const auto result11 = SelectCoins(*wallet, available_coins, /*pre_set_inputs=*/{}, 10 * CENT - fee_to_deduce, coin_control, coin_selection_params_bnb);
```

Comment:
Nit: I think "deduct" would fit better here than "deduce":

```suggestion
        CAmount fee_to_deduct = available_coins.coins[OutputType::BECH32].begin()->GetFee();
        const auto result11 = SelectCoins(*wallet, available_coins, /*pre_set_inputs=*/{}, 10 * CENT - fee_to_deduct, coin_control, coin_selection_params_bnb);
```

Reply:
Done as suggested